composer.json

```diff
@@ -9,14 +9,14 @@
 	],
 	"type": "cakephp-plugin",
 	"require": {
-		"cakephp/core": "~3.5",
+		"cakephp/core": "^3.6.0",
```

Comment:
Do we need to make this +3.6 only plugin?

Reply:
We do if we want to use `getRepository()` to avoid the deprecation warning for `repository()`.

Reply:
Either silence or if/else switch based on version.